### PR TITLE
Add increment value attribute on Duplicate Device Name for Route Report.

### PR DESCRIPTION
**Pull request by LiGuru**
[fix] Excel limitation workaround for duplicate Sheets names
[Forum discussion](https://www.traccar.org/forums/topic/report-route-if-there-are-duplicate-device-names/)
[Expected result](https://github.com/traccar/traccar/files/11714435/report-110623135938.xlsx)

Declaring uniqueSheetsTitle
Add private method getUniqueSheetName
Usage in getExcel sheetNames

### CODE REVIEW

File: src/main/java/org/traccar/reports/RouteReportProvider.java [hunk 1]
```diff
@@ -66,7 +69,13 @@ public Collection<Position> getObjects(long userId, Collection<Long> deviceIds,
         }
         return result;
     }
+    private final Map<String, Integer> uniqueSheetsTitle = new HashMap<>();
 
+    private String getUniqueSheetName(String key) {
+        AtomicInteger atomic = new AtomicInteger(uniqueSheetsTitle.getOrDefault(key, -1));
+        uniqueSheetsTitle.put(key, atomic.incrementAndGet());
+        return (uniqueSheetsTitle.get(key) > 0 ? key + "-" + uniqueSheetsTitle.get(key) : key);
```
Comment (tananaev) [inline src/main/java/org/traccar/reports/RouteReportProvider.java:77]: Why do you need `AtomicInteger`? Also I think the numbers should start with 1 and not -1. So basically if there's 1 device, we don't show. If more than 1, we do "Name", "Name-2", "Name-3" etc.

File: src/main/java/org/traccar/reports/RouteReportProvider.java [hunk 2]
```diff
@@ -66,7 +69,13 @@ public Collection<Position> getObjects(long userId, Collection<Long> deviceIds,
         }
         return result;
     }
+    private final Map<String, Integer> uniqueSheetsTitle = new HashMap<>();
```
Comment (tananaev) [inline src/main/java/org/traccar/reports/RouteReportProvider.java:72]: Please move to the top next to other variables. And also name doesn't really make sense. It just contains the number of times a name is included. Maybe just call `namesCount` or something like that.

**Comment by LiGuru** [comment]
I use AtomicInteger because of NullPointerException and adding an element with default value. 

Less code I guess.

If you suggest another implementation I will consider it.

**Comment by tananaev** [comment]
Why would there be a `NullPointerException`?

As for default, you should use `computeIfAbsent`.

**Comment by LiGuru** [comment]
What about putIfAbsent instead of computeIfAbsent
Is a lambda/function needed here?

`putIfAbsent(key, 0);`
`computeIfPresent(key, (k, v) -> v + 1);`

**Comment by tananaev** [comment]
Actually maybe you just use compute like this:
```
map.compute(key, (k, value) -> (value == null) ? 1 : (value + 1))
```

**Comment by LiGuru** [comment]
I'm thinking should it be implemented in all reports where Excel Sheets are needed?

**Comment by tananaev** [comment]
Probably in all reports that use tabs.

**Pull request merged**